cl/recap/mergers.py

```diff
@@ -1382,13 +1386,52 @@ def merge_pacer_docket_into_cl_docket(
         add_docket_entries
     )(d, docket_data["docket_entries"], tags=tags)
     add_parties_and_attorneys(d, docket_data["parties"])
-    process_orphan_documents(rds_created, d.court_id, d.date_filed)
+    async_to_sync(process_orphan_documents)(
+        rds_created, d.court_id, d.date_filed
+    )
     logger.info(f"Created/updated docket: {d}")
     return rds_created, content_updated
 
 
-@transaction.atomic
-def merge_attachment_page_data(
+async def clean_duplicate_attachment_entries(
+    de: DocketEntry,
+    document_number: int,
+    attachment_dicts: List[Dict[str, Union[int, str]]],
+):
+    """Remove attachment page entries with duplicate pacer_doc_id's that
+    have incorrect attachment numbers.
```

Comment:
```suggestion
    """Remove attachment page entries with duplicate pacer_doc_id's that
    have incorrect attachment numbers. This is needed because older attachment
    pages were incorrectly parsed. See: freelawproject/juriscraper#721
```